content/components/action-list.mdx

```diff
@@ -0,0 +1,157 @@
+---
+title: Action list
+status: Alpha
+---
+
+import {Grid, Flex, Box, Button, ButtonOutline, Heading} from '@primer/components'
+
+<Box sx={{fontSize: 3}} class="lead" as="p">
+    An action list is a vertical list of interactive actions or options. It’s composed of items presented in a consistent single-column format, with room for icons, descriptions, side information, and other rich visuals.
+</Box>
+
+<img src="https://user-images.githubusercontent.com/293280/122133975-96c1e000-cdf2-11eb-9ce9-02d5014c1831.png" alt="Action list examples" />
+
+<!--
+<ButtonOutline mr={1}>React component</ButtonOutline>
+<ButtonOutline mr={1}>ViewComponent</ButtonOutline>
+<ButtonOutline>Figma component</ButtonOutline>
+-->
+
+## Overview
+
+Action lists can have many applications. They’re the foundation of menus, selection panels, and other overlay components; they can be applied to page sidebars for showing individual actions, for handling local navigation, and for displaying metadata.
+
+Action lists support section dividers and headers for grouping items, and indivudal item dividers for added clarity.
+
+Action lists use a mobile-friendly inset style. Their sizes are adapted on touch devices, and their single-column format should render consistently in any screen size.
```

Comment:
Could it be:
Their size adapts to touch devices.

Or:
Their size adjusts to touch devices.

Reply:
```suggestion
Action lists use a mobile-friendly inset style. Their sizes are adapted on touch devices, and their single-column format renders consistently in any screen size.
```